polyfill.yml

```diff
@@ -0,0 +1,40 @@
+base: luau
```

Comment:
The polyfill is in the root so that you can run selene with the CLI or VSCode extension if you wish